core/controllers/editor_test.py

```diff
@@ -175,34 +190,40 @@ def _put_and_expect_400_error(payload):
         self.logout()
 
     def test_resolved_answers_handler(self):
```

Comment:
FYI: This test has been removed in the answer migrations branch, since resolved answers are obsolete.